templates/repo/header.tmpl

```diff
@@ -61,6 +61,16 @@
 							</div>
 						</form>
 					{{end}}
+					{{if $.Permission.IsAdmin}}
+					<form method="post" action="{{$.RepoLink}}/action/{{if .IsPinned}}un{{end}}pin?redirect_to={{$.Link}}">
+						{{$.CsrfTokenHtml}}
+						<div class="ui labeled button" tabindex="0">
+							<button type="submit" class="ui compact small basic button"{{if not $.IsSigned}} disabled{{end}}>
```

Comment:
```suggestion
							<button type="submit" class="ui compact small basic button">
```
I simply don't see the value in showing a disabled button for anonymous users. Let's just leave it out completely.